src/consts/sorts.js

```diff
@@ -0,0 +1,2 @@
+export const HTL = "High to low";
```

Comment:
I meant the string value can be a short one like HTL, but the const name need to be understandable like HIGH_TO_LOW